### DISCUSSION TITLE
THE CONTENTS RETURNS TO IT'S ORIGINAL FORM AFTER MAKING CHANGES AND RUN bundle exec jekyll serve

### DISCUSSION THREAD

**Discussion by innocent-charles** [alshedivat/al-folio#703] [Q&A]
I'm trying to change /_site/ index.html so as to customize this theme on my own view . When I edit the index.html and start again  , index.html back to it's original content.please help me out !

**Comment by alshedivat** [comment]
`/_site/ index.html` is not supposed to be changed when you work with jekyll. this happens because once you run jekyll, it rebuilds everything inside `/_site/*` from scratch.

if you want to customize some content on posts or pages, then edit files in `_posts/*` or `_pages/*`, respectively. if you want to customize layouts, you would need to edit files in `_layouts/*`. if you want to customize styles, then edit files in `_sass/*`.
